src/Controls/src/Core/ContentPage/ContentPage.Android.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.Maui.Controls
+{
+	public partial class ContentPage
+	{
+		static void MapInputTransparent(IPageHandler handler, ContentPage page)
```

Comment:
Did you test on iOS? If is working on iOS, https://github.com/dotnet/maui/blob/a11555b9c8dfd7a80f89917eab7b9558c442bfff/src/Core/src/Handlers/View/ViewHandler.cs#L495 should works, and need to review if in Android somewhere we are overriding the behavior.

Reply:
iOS works as expected without any changes needed

https://github.com/user-attachments/assets/220756a9-461f-47ed-b893-254d71ebfad8

^no crash yay